tests/queries/0_stateless/03172_error_log_table_not_empty.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+# Tags: no-parallel
```

Comment:
Let's not add `no-parallel`. It takes a lot of effort to remove them later. (https://github.com/ClickHouse/ClickHouse/pull/65186) The test will work fine if executed in parallel and we should be fine if the error count is higher.